Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -748,6 +757,11 @@ class SettingsTableSectionHeaderFooterView: UITableViewHeaderFooterView {
                 make.top.equalTo(self).offset(SettingsTableSectionHeaderFooterViewUX.titleVerticalLongPadding)
             }
         }
+        
+        titleLabel.snp.remakeConstraints({ make in
```

Comment:
We should try to set the constraints correctly in the switch statement above instead of remaking the constraints here again. 
If the constraints are remade here all the constraints set above are never applied.